src/components/NavBarItem/navBarItem.module.css

```diff
@@ -0,0 +1,28 @@
+.listItem {
+  list-style-type: none;
+  font-family: Mada;
+}
+
+.link {
```

Comment:
Is this general, put into generic class to reuse.

Reply:
Fixed